fvm/meter/weighted_meter.go

```diff
@@ -368,6 +386,13 @@ func (m *WeightedMeter) MergeMeter(child Meter) {
 	for key, intensity := range child.MemoryIntensities() {
 		m.memoryIntensities[key] += intensity
 	}
+
+	// merge storage meters
+	for key, value := range child.StorageUpdateSizeMap() {
+		m.storageUpdateSizeMap[key] += value
```

Comment:
They should not be summed. Only take the child value.

Reply:
addressed.